feature-libs/product-configurator/package.json

```diff
@@ -33,13 +33,13 @@
     "@ng-select/ng-select": "^10.0.3",
     "@ngrx/effects": "^15.3.0",
     "@ngrx/store": "^15.3.0",
-    "@spartacus/cart": "5.0.0",
-    "@spartacus/checkout": "5.0.0",
-    "@spartacus/core": "5.0.0",
-    "@spartacus/order": "5.0.0",
-    "@spartacus/schematics": "5.0.0",
-    "@spartacus/storefront": "5.0.0",
-    "@spartacus/styles": "5.0.0",
+    "@spartacus/cart": "6.5.0-1",
+    "@spartacus/checkout": "6.5.0-1",
+    "@spartacus/core": "6.5.0-1",
+    "@spartacus/order": "6.5.0-1",
+    "@spartacus/schematics": "6.5.0-1",
+    "@spartacus/storefront": "6.5.0-1",
+    "@spartacus/styles": "6.5.0-1",
```

Comment:
[QUESTION]
Is this change _really_ related to this PR?
(I don't mind it, as it's just a partial PR to an epic branch. But just wanted to understand and dbl-check it)